docs/deployment.md

```diff
@@ -66,31 +66,41 @@ Examples, at the time of writing, of the typical case for these are:
 
 ### Environment variables for deployment
 
-The `$DEPLOY_ENV` file defines some *actual* environment variables
+The `$DEPLOY_ENV` file defines some _actual_ environment variables
 which the deployment process will use. What those are set to will
-depend on your situation, so this is just a guide. 
+depend on your situation, so this is just a guide.
 
-An example of the contents of a `$DEPLOY_ENV` file, but with secret
-values redacted:
+Here is an example of how the `$DEPLOY_ENV` file may be created as the
+application user, but with secret values redacted:
 
+    cat > $DEPLOY_ENV <<EOF
     export USERDIR=/home/$USER
-    export DEPLOY_DEST=$USERDIR/deploy
+    export GIT_WORKING=/home/$USER/gitworking/mykomap-monolith
+    export DEPLOY_DEST=/home/$USER/deploy
```

Comment:
Note that in general Linux does not require home directories to be anywhere in particular. This is why I avoided hardwiring the `DEPLOY_DEST` directory to `/home/$USER/deploy` which makes the implicit assumption they'll be in `/home`, and put `$USERDIR/deploy` instead. (I am avoiding assuming a DCC server is being used as well)